docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -33,7 +33,9 @@ If you would like a reliable, scalable and fully managed Metabase, please consid
 
 ## Quick Launch
 
-Download the [Metabase AWS source bundle file](https://downloads.metabase.com/{{ site.latest_version }}/metabase-aws-eb.zip) to upload to Elastic Beanstalk.
+Download the [Metabase Community Edition AWS source bundle file](https://downloads.metabase.com/{{ site.latest_version }}/metabase-aws-eb.zip) to upload to Elastic Beanstalk. 
+
+If you are looking to deploy our **Enterprise Edition**, you need to go to the link https://downloads.metabase.com/enterprise/**version**/metabase-aws-eb.zip. replacing the **version** part of the URL with the version that you want to deploy (e.g v1.38.3).
```

Comment:
```suggestion
Download the [Metabase Open Source Edition AWS source bundle file](https://downloads.metabase.com/{{ site.latest_version }}/metabase-aws-eb.zip) to upload to Elastic Beanstalk. 

For the **Enterprise Edition**, visit 
https://downloads.metabase.com/enterprise/**version**/metabase-aws-eb.zip,

replacing the **version** part of the URL with the version that you want to deploy (e.g., v1.38.3).
```

Is there instead a landing page we could direct them to, rather than ask people to choose a version and compose a URL?

Reply:
Unfortunately no, I'll delete these lines and if the customers ask, we'll send it through the support channel